site/src/atomics/width.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Width Atomics
+description: Width related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Width Atomics
+
+Need to specifiy a specific width for an element? You've come to the right place! These classes should be used sparingly, and only used after testing heavily at all screen widths (starting with mobile). That said, if you need to specify and min, max or normal width, they can be a useful tool in the utility belt.
+
+Values are in pixels. Keep in mind that the `min-width` and `max-width` properties override the standard `width` property. If you aren't using the Atlas core folder as a base for you styles, these classes work best when [you're setting `box-sizing` to be `border-box` in some way](https://css-tricks.com/box-sizing/#aa-present-day-box-sizing).
+
+| cssproperty                                      | value (px)                                       | screensize                            |
+| ------------------------------------------------ | ------------------------------------------------ | ------------------------------------- |
+| `width`                                          | `auto`, `100`, `150`, `200`, `250`, `300`, `350` | all screensizes, `tablet`, `desktop`, |
+| `width` (available on tablet screens and larger) | `400`, `450`, `unset`                            | `tablet`, `desktop`,                  |
+
+## Usage
+
+Set the `width` of an element.
+
+```html
+<div class="width-auto border padding-sm font-weight-semibold">
+	<p>Auto</p>
+</div>
+<div class="width-100 border padding-sm font-weight-semibold">
+	<p>100</p>
+</div>
+<div class="width-150 border padding-sm font-weight-semibold">
+	<p>150</p>
+</div>
+<div class="width-200 border padding-sm font-weight-semibold">
+	<p>200</p>
+</div>
+<div class="width-250 border padding-sm font-weight-semibold">
+	<p>250</p>
+</div>
+<div class="width-300 border padding-sm font-weight-semibold">
+	<p>300</p>
+</div>
+<div class="width-350 border padding-sm font-weight-semibold">
+	<p>350</p>
+</div>
+<div class="display-flex align-items-center padding-sm color-danger font-size-sm">
+	<div class="flex-grow-1 margin-right-xs border-color-danger border-top"></div>
+	<p>The following are only applied above the tablet breakpoint</p>
+	<div class="flex-grow-1 margin-left-xs border-color-danger border-top"></div>
+</div>
+<div class="width-400-tablet border padding-sm font-weight-semibold">
+	<p>400</p>
+</div>
+<div class="width-450-tablet border padding-sm font-weight-semibold">
+	<p>450</p>
+</div>
+```
+
+### Responsive rules
+
+Certain widths are too large for many mobile screens, so they are only available on larger screensizes. However, all helpers can be applied to any screensize.
+
+```html
+<div class="width-250 width-300-tablet width-400-desktop border padding-sm">
+	<p>250 ➡ 300 ➡ 350 ➡ 450</p>
```

Comment:
Remove `➡ 350`, update `450` to `400` to match what's actually applied.

Reply:
🤦‍♂️gets me every time!

Reply:
updated